modules/stanford_profile_styles/webpack.config.js

```diff
@@ -46,6 +46,8 @@ var webpackConfig = {
   devtool: 'source-map',
   // What build?
   entry: {
+    "admin": path.resolve(__dirname, srcSass + "/admin/index.scss"),
+    "field_widgets": path.resolve(__dirname, srcSass + "/admin/stanford_profile_styles.field_widgets.scss"),
```

Comment:
@pookmish,

This is creating a file "field_widgets.css"
Is this what you're asking for?

Reply:
You've got a disconnect here. Your library is looking for a CSS file called `stanford_profile_styles.field_widgets.css` but you are outputting `field_widgets.css` so your styles are not loading. You need to change the key in this object to `stanford_profile_styles.field_widgets` instead of `field_widgets` to get the correct name. The key in the entry object is what the filename will be.